internal/venice-common/src/main/java/com/linkedin/venice/meta/Version.java

```diff
@@ -324,6 +329,29 @@ static boolean isStreamReprocessingTopic(String kafkaTopic) {
     return checkVersionSRTopic(kafkaTopic, true);
   }
 
+  static boolean isChangeCaptureTopic(String kafkaTopic) {
+    return checkVersionChangeCaptureTopic(kafkaTopic, true);
```

Comment:
Is this the only caller to `checkVersionChangeCaptureTopic()` or you envision more callers? If this the only one, can we remove the second params here and in the function's definition as it is always true?

Reply:
We might have more calllers in the future, as server or controller needs to check change capture topic similar to stream reprocessing topic.